packages/edit-post/src/components/options-modal/index.js

```diff
@@ -125,6 +129,8 @@ export default compose(
 	withDispatch( ( dispatch ) => {
 		return {
 			closeModal: () => dispatch( 'core/edit-post' ).closeModal(),
+			toggleIconLabels: () =>
+				dispatch( 'core/edit-post' ).toggleFeature( 'showIconLabels' ),
```

Comment:
Doesn't look like this is used anywhere?

Reply:
Whoops, refactored and forgot to remove it 😅